### DISCUSSION TITLE
Add minified string when looting gold as Zandalari Troll

### DISCUSSION THREAD

**Discussion by michalsniezko** [kaytotes/ImprovedBlizzardUI#184] [Ideas]
When playing as a Zandalari Troll the message about looting money looks different from other races and I think that's the reason its not minified:
```
+ [Pointy Stick]
You loot 84 Silver, 28 Copper (+1 Silver, 68 Copper)
You loot 1 Gold, 35 Silver, 4 Copper (+2 Silver, 70 Copper)
+ [Shrouded Cloth] (2)
You loot 1 Gold, 16 Silver, 49 Copper (+2 Silver, 32 Copper)
```

**Comment by kaytotes** [comment]
I've converted this to an issue and should have it fixed in the next release. I've had to push out a fair few fixes this past few days which can be annoying for users so will likely wait until the next "major" release. However please note I've identified the issue. 

There are a few other missing strings I plan to hunt down too.
